packages/wp-story-editor/src/components/documentPane/status/status.js

```diff
@@ -37,6 +48,37 @@ import {
  */
 import { VISIBILITY, STATUS } from '../../../constants';
 
+const StyledListItem = styled.li`
+  position: relative;
+  padding: 6px 8px;
+  margin: 4px 8px;
+  align-items: center;
+  border-radius: 4px;
+  cursor: pointer;
+  ${themeHelpers.focusableOutlineCSS}
```

Comment:
💭 nit but may not be necessary:
```suggestion
  ${themeHelpers.focusableOutlineCSS};
```

Reply:
whoops. Thanks.

Reply:
I don't think this semicolon is needed 🤔

Reply:
`${themeHelpers.focusableOutlineCSS}` does not have a semicolon anywhere else in our code base either